src/helper/isCtrlEnter.ts

```diff
@@ -0,0 +1,13 @@
+import { KeyboardEvent } from "react";
+
+/**
+ * Checks if given keyboard event is Ctrl + Enter or Cmd + Enter
+ * @param e Keydown event
+ * @returns true if Ctrl + Enter (on windows) or Cmd + Enter (on mac) is pressed
+ */
+export default function isCtrlEnter(e: KeyboardEvent<HTMLInputElement | HTMLTextAreaElement>) {
+  if ((e.ctrlKey || e.metaKey) && e.key === "Enter") {
```

Comment:
is this handled for MacOS's command key?

Reply:
yep. metaKey is equivalent to cmd key in mac os

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

Reply:
ok, so we are good to go